src/pages/index.js

```diff
@@ -49,35 +65,115 @@ function handleImageClick(data) {
 }
 
 function handleEditSumbit(obj) {
+  popupEdit.renderLoading(true);
   userInfo.setUserInfo(obj);
-  popupEdit.close();
-  formEditValidator.disableButton();
+  api
+    .updateCurrentUser(
+      JSON.stringify({
+        name: obj.name,
+        about: obj.about,
+      })
+    )
+    .then(() => {
+      popupEdit.close();
+      popupEdit.renderLoading(false);
+      formEditValidator.disableButton();
+    });
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**